java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -183,4 +183,42 @@ CompletableFuture<Double> zaddIncr(
      * }</pre>
      */
     CompletableFuture<Long> zcard(String key);
+
+    /**
+     * Removes and returns up to <code>count</code> members with the highest scores from the sorted
+     * set stored at the specified <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/zpopmax/">redis.io</a> for more details.
+     * @param key The key of the sorted set.
+     * @param count Specifies the quantity of members to pop.<br>
+     *     If <code>count</code> is higher than the sorted set's cardinality, returns all members and
+     *     their scores, ordered from highest to lowest.
+     * @return A map of the removed members and their scores, ordered from the one with the highest
+     *     score to the one with the lowest.<br>
+     *     If <code>key</code> doesn't exist, it will be treated as an empty sorted set and the
+     *     command returns an empty <code>Map</code>.
+     * @example
+     *     <pre>{@code
+     * Map<String, Double> payload = client.zpopmax("mySortedSet", 2).get();
+     * assert payload.equals(Map.of('member2', 8.0, 'member3', 7.5)); // Indicates that 'member1' with a score of 10.0 has been removed from the sorted set.
```

Comment:
Where is member1 defined in this example?

Reply:
These examples are taken from Node and Py clients. This is just high level overview on how the command can be used if the user wants detailed info they can check redis as well.